ci/scripts/quick-bazel.sh

```diff
@@ -8,7 +8,82 @@
 # This doesn't install dependencies, but should otherwise behave the
 # same as what CI would do on a pull request
 
-bazel query "rdeps(//..., //hw:verilator)" | \
-            sed -e 's/^/-/' | \
-            xargs bazel test --nobuild_tests_only --test_tag_filters=-cw310,-verilator -- //...
+set -e -x
+
+DEFAULT="master"
+
+git config advice.detachedHead false
+
+ORIGINAL_BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD 2>/dev/null || echo "")
+ORIGINAL_SHA=$(git rev-parse HEAD)
+
+mkdir -p bazel-results
+
+# If a test is moved or intentionally broken, tagging it lets CI know to skip
+# it when assessing the pr and the merge base.
+bazel query 'attr("tags", "broken", tests(//...))' > bazel-results/ci_run_ignore.txt
```

Comment:
I don't think I understand this: aren't we just going to get everything clean?

Reply:
@cfrantz has justified the broken tag in the discussion of this PR, and I found it convincing. Having a single test defeature isn't abnormal, and since reverting a commit takes an hour to clear CI, this gives people an opportunity to make considered/reviewed methods to clear CI if necessary.

re: Getting everything clean: [[bazel] Add bazel to lists of required packages and to CI](https://github.com/lowRISC/opentitan/pull/10041) did just that, after which I was asked to minimize impact on parties unfamiliar with Bazel, and this is a lower impact way to get there.

Reply:
Ok, I understand. I hope we can remove this once we've migrated from Meson properly: long-term, I'd expect something that runs in CI to pass.

Reply:
The broken tag is a mechanism to stop things from running in CI. At present nothing is tagged as broken. I'm happy to help replace this script with something more robust in the future.

When discussing a replacement we should consider how to keep most innocent PRs unblocked in our CI when post merge tests are made to fail. A good answer to that question makes this all obsolete.

Reply:
This probably isn't the right venue to have this discussion but... as I see it the *whole point* of CI is to block PRs if something is broken. Merge skew is annoying and can mess stuff up, but if a test is broken then we *should* be running around like our hair is on fire until it's fixed.

Reply:
I agree.  Accepting a test as broken should be both deliberate and temporary.  Tagging a broken test in the build configuration and requiring a commit satisfies the deliberate part of that requirement.  Excepting those tests which are deliberately skipped, we should be executing all tests.